ros_speech_recognition/scripts/speech_recognition_node.py

```diff
@@ -322,11 +392,14 @@ def speech_recognition_srv_cb(self, req):
                 rospy.loginfo("Waiting for result... (Sent %d bytes)" % len(audio.get_raw_data()))
 
                 try:
+                    header = std_msgs.msg.Header(stamp=rospy.Time.now())
```

Comment:
I think this timestamp may be misleading. (Not start time of original audio data, But the time of speech recognition).